db/schema.rb

```diff
@@ -64,25 +64,26 @@
     t.index ["record_id"], name: "index_record_data_on_record_id"
   end
 
-  create_table "records", id: :serial, force: :cascade do |t|
-    t.string "recordable_type", null: false
+  create_table "records", force: :cascade do |t|
     t.integer "recordable_id", null: false
+    t.string "recordable_type", null: false
     t.string "revision_id", null: false
     t.datetime "closed_date"
     t.string "format", default: ""
     t.string "state"
     t.string "associated_granule_value"
     t.string "copy_recommendations_note"
+    t.datetime "state_last_modified"
```

Comment:
searching on the page, I don't see this used or the migration that added this.

Reply:
good catch, yes this needs to be removed, was. a change I made a while back in testing that need to be reverted.